eisbuk-admin/src/components/atoms/UserAttendance/UserAttendance.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemAvatar from "@material-ui/core/ListItemAvatar";
+import ListItemSecondaryAction from "@material-ui/core/ListItemSecondaryAction";
+import ListItemText from "@material-ui/core/ListItemText";
+import EisbukAvatar from "@/components/users/EisbukAvatar";
+import { Customer, BookingsMeta } from "eisbuk-shared";
+import Button from "@material-ui/core/Button";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import { ETheme } from "@/themes";
+import { markAttendance } from "@/store/actions/attendanceOperations";
+import { useDispatch } from "react-redux";
+
+type UserBooking = BookingsMeta & Pick<Customer, "certificateExpiration">;
+
+interface Props {
+  isAbsent?: boolean;
+  userBooking: UserBooking;
+  toggleAbsent: (UserBooking: UserBooking, isAbsent: boolean) => void;
+  hasLocalChange: boolean;
+}
+const UserAttendance: React.FC<Props> = ({
+  isAbsent,
+  userBooking,
+  toggleAbsent,
+  hasLocalChange,
```

Comment:
What you need here is: `customerId`, `customerName`, `attended`, `bookingId`, nothing more, nothing less